[EDU-5669] feature: add and publish Savings Plan Terms and Conditions (#1382)
aziontech/docs: src/content/docs/en/pages/agreements/savings-plan/savings-plan.mdx

---
title: Savings Plan Terms and Conditions

> description: Terms and Conditions of the Savings Plan

Author: ```suggestion
description: Terms and Conditions related to the Savings Plan
```

meta_tags: Terms and Conditions of the Savings Plan, Azion 
namespace: docs_agreements_terms_and_conditions_savings_plan
permalink: /documentation/agreements/terms-and-conditions-savings-plan/
---

> Savings Plan is one of the ways of contracting Azion Products and Services, subject to the terms of the [Customer Agreement](https://www.azion.com/en/documentation/agreements/customer-agreement/) and other related documents. In this format, the Customer commits to pay a financial amount during the Effective Term (the "Purchase Commitment") and can enjoy discounts on On Demand Prices published on the subscription date, in accordance with their Service Order ("On Demand Price Discount"), except on third-party products listed on the Azion Marketplace.

Author: ```suggestion
Savings Plan is one of the ways of contracting Azion Products and Services, subject to the terms of the [Customer Agreement](/en/documentation/agreements/customer-agreement/) and other related documents. In this format, the Customer commits to pay a financial amount during the Effective Term (the "Purchase Commitment") and can enjoy discounts on On Demand Prices published on the subscription date, in accordance with their Service Order ("On Demand Price Discount"), except on third-party products listed on the Azion Marketplace.
```

## 3. Other Terms

3.1. In the event of non-compliance with any clause of the Customer Agreement, as well as any other contract or term agreed upon between the parties, including the Service Order, the Customer shall be subject to the disregard of the Discount on the On Demand Price and retroactive billing from the beginning of the Term of Validity of the amounts recalculated as On Demand consumption.

3.2. The Customer is aware and agrees that the Products and Services will be subject to the On Demand Prices, billed monthly, when: (a) their consumption exceeds the Purchase Commitment and any credits acquired or received; (b) using Products and Services not eligible for the Discount on the On Demand Price or that are not specified in the Service Order; or (c) after the Term of Validity, there is no automatic renewal or signing of a new Service Order.

3.3. In the event of a conflict or divergence between the conditions defined herein and in the Service Order, the provisions of the Service Order shall prevail.

## 4. Definitions

> 4.1 The following are added to the Terms and Definitions of the Customer Agreement:

Author: A numer

> 4.1 The following are added to the Terms and Definitions of the Customer Agreement:

Author: ```suggestion
4.1. The following are added to the Terms and Definitions of the Customer Agreement:
```
